app/src/main/java/org/mozilla/fenix/addons/AddonDetailsView.kt

```diff
@@ -102,10 +105,34 @@ class AddonDetailsView(
         val parsedText = detailsText.replace("\n", "<br/>")
         val text = HtmlCompat.fromHtml(parsedText, HtmlCompat.FROM_HTML_MODE_COMPACT)
 
-        details.text = text
+        val spannableStringBuilder = SpannableStringBuilder(text)
+        val links =
+            spannableStringBuilder.getSpans(0, text.length, URLSpan::class.java)
```

Comment:
Let's use the KTX version here
```suggestion
            spannableStringBuilder.getSpans<URLSpan>()
```